src/index.js

```diff
@@ -1,9 +1,11 @@
 import assign from 'object-assign';
 import pascalCase from 'pascal-case';
+import React from 'react';
```

Comment:
@bradleyayers the build will generate a file in dist that has a UMD definition. This means it will resolve correctly in all environments as long as the dependency is there (either NPM or <script> tag).